Cue/app/tabs/library/LibraryListView.js

```diff
@@ -3,30 +3,122 @@
 'use strict';
 
 import React from 'react'
-import { Navigator, View, Text, TouchableOpacity } from 'react-native'
+import { Navigator, View, Text, ListView, Dimensions } from 'react-native'
 
-import CueColors from '../../common/CueColors'
+import ListViewHeader from '../../common/ListViewHeader'
+import LibraryListViewItem from './LibraryListViewItem'
+import LibraryEmptyView from './LibraryEmptyView'
+
+// TODO These typedefs should be moved to a common (API?) class
+type Card = {
+  uuid: string,
+  front: string,
+  back: string,
+  needsReview: boolean,
+  position: number
+}
+
+type Deck = {
+  uuid: string,
+  name: string,
+  rating: number,
+  numRatings: number,
+  tags: Array<string>,
+  owner: string,
+  public: boolean,
+  deckVersion: number,
+  userDataVersion: number,
+  created: Date,
+  lastUpdate: Date,
+  lastUpdateDevice?: string,
+  shareCode?: string,
+  deleted: boolean,
+  cards: Array<Card>
+}
```

Comment:
Agreed these types should be moved to a common class. api/types.js would work. This way the api can easily use and return these types